package.json

```diff
@@ -28,6 +28,7 @@
     "eslint": "^3.17.1",
     "eslint-config-taskcluster": "^2.0.0",
     "eslint-plugin-taskcluster": "^1.0.0",
+    "lodash": "^4.17.4",
```

Comment:
Let's try to avoid this... it's a bit risky as tests would pass if in the future we ever accidentally started to use it in the package..